django_server/.env.sample

```
@@ -0,0 +1,80 @@
+EMAIL_USE_TLS=True
+EMAIL_HOST_USER=your-email@gmail.com
+EMAIL_HOST_PASSWORD=your-email-password
+
+# Logging
+LOG_LEVEL=INFO
+LOG_FILE=/workspace/django_server/logs/anythingllm.log
+
+# Telemetry
+TELEMETRY_ENABLED=False
+POSTHOG_API_KEY=your-posthog-api-key
+POSTHOG_HOST=https://app.posthog.com
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)